### ISSUE TITLE
pygithub3 still used - sync_repos broken

### ISSUE THREAD

**Issue by pypingou** [ralphbean/pep8bot#28]
The commit https://github.com/ralphbean/pep8bot/commit/552ae1eb63799acd82ddfb206e6d6d535885db07 removed the dependency to pygithub3 which fixed #26 but now the sync_repos method is broken: https://github.com/ralphbean/pep8bot/blob/develop/pep8bot/models/__init__.py#L65-L72

**Comment by pypingou** [comment]
Looks like https://github.com/ralphbean/pep8bot/blob/develop/pep8bot/models/__init__.py#L90 is also affected

**Comment by pypingou** [comment]
Note: github lists a number of python library for interacting with their API: https://developer.github.com/libraries/
- python-github3 https://github.com/copitux/python-github3/graphs/contributors pretty much dead since October 2012
- github3.py https://github.com/sigmavirus24/github3.py/graphs/contributors seems alive
- githubpy https://github.com/michaelliao/githubpy/graphs/contributors seems alive but has much less contributions
- PyGithub https://github.com/jacquev6/PyGithub/graphs/contributors seems to be surviving
- libsaas https://github.com/ducksboard/libsaas/graphs/contributors 9 contributors and seems to be surviving
- sanction https://github.com/demianbrecht/sanction/graphs/contributors 5 contributors and does not seems alive
- agithub https://github.com/jpaugh/agithub/graphs/contributors 4 contributors and seems a bit dead
- octohub https://github.com/turnkeylinux/octohub/graphs/contributors 2 contributors and seems dead
- torngithub https://github.com/jkeylu/torngithub/graphs/contributors 1 contributor and seems dead

So if we want to reconsider the github library, I guess I'd recommend switching to github3.py

**Comment by ralphbean** [comment]
I agree.  https://github.com/sigmavirus24/github3.py/graphs/contributors looks best.
